ops/test_fullstack.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
```

Comment:
We can do something fancier in the future (e.g. use [Puppeteer](https://github.com/puppeteer/puppeteer) to test interactive stuff rather than just use requests which misses if JS crashes or doesn't load.